packages/base-styles/_mixins.scss

```diff
@@ -428,6 +428,14 @@
 	// Hexadecimal css vars do not work in the rgba function.
 	--wp-admin-theme-color: #{$color-primary};
 	--wp-admin-theme-color--rgb: #{hex-to-rgb($color-primary)};
+
+	// Lighter shades.
```

Comment:
I'm thinking, the easier path forward would be to use `rgba(var(--wp-admin-theme-color--rgb), 0.08)` directly. 

Example:

```css
  --wp-admin-theme-color-opaque-04:rgba(var(--wp-admin-theme-color--rgb), 0.04);
  --wp-admin-theme-color-opaque-08:rgba(var(--wp-admin-theme-color--rgb), 0.08);
```

This means no color changes, but the vars a bit clunky.